assets/src/scss/layout/_featured-action.scss

```diff
@@ -0,0 +1,86 @@
+.featured-action-section {
+  margin-top: $sp-6x;
+  margin-bottom: $sp-6x;
+  display: flex;
+  border-radius: $sp-x;
+  gap: $sp-3;
+  box-shadow: 0 1px 8px rgba(0, 0, 0, .16);
```

Comment:
After the new identity release, we've replaced all `hexa` or `rgba` colors to `$grey-900`. Then In this case just set as `rgba($grey-900, .16)`.

Reply:
I talked with Houssam about this, and we are trying to maintain consistency with the [TakeActionCovers](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks/Covers/styles/TakeActionCovers.scss#L18), which also has the same `box-shadow `value